products/satellite.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Red Hat Satellite Server
+category: server-app
+changelogTemplate: "https://access.redhat.com/documentation/en-us/red_hat_satellite/__RELEASE_CYCLE__/html/release_notes/index"
+releases:
+-   releaseCycle: "6.12"
+    eol: 2024-05-31
+    support: 2023-05-31
+    releaseDate: 2022-11-16
+    latest: "6.12.0"
+    latestReleaseDate: 2022-11-16
+-   releaseCycle: "6.11"
+    eol: 2024-01-31
+    support: 2022-11-30
+    releaseDate: 2022-07-05
+    latest: "6.11.4.1"
+    latestReleaseDate: 2022-11-17
+-   releaseCycle: "6.10"
+    eol: 2023-05-31
+    support: 2022-06-30
+    releaseDate: 2021-11-16
+    latest: "6.10.7.1"
+    latestReleaseDate: 2022-11-17
+-   releaseCycle: "6.9"
+    eol: 2022-11-30
+    support: 2021-11-30
+    releaseDate: 2021-11-16
+    latest: "6.9.10"
+    latestReleaseDate: 2022-11-17
+iconSlug: redhat
+permalink: /satellite
+alternate_urls:
+-   /rhsat
+releasePolicyLink: https://access.redhat.com/support/policy/updates/satellite
+activeSupportColumn: Full support
+releaseColumn: true
+releaseDateColumn: true
+eolColumn: Maintenance support
+discontinuedColumn: false
+versionCommand: yum info satellite
```

Comment:
just a small question : wasnt redhat using dnf ? i mean shouldnt this one be `dnf info satellite` "

Reply:
Thank you for fixing the Issues.
Red Hat has supported dnf since RHEL 8, but the version still supported, Satellite 6.10, is only compatible with RHEL 7. yum can be used under both versions of Red Hat Enterprise Linux, so I thought this was the more general approach.

Reply:
i see , how about wont this will show upcoming info for maybe a not released version instead of showing current version ?
i mean isnt there a better way to show current version such as `cat /etc/redhat-release`

Reply:
i see

Reply:
Not any that I know about.
In this [Red Hat Article](https://access.redhat.com/solutions/2159361) it is discribed with `rpm -q satellite --queryformat "%{VERSION}"` for Version 6.

The installed Version is also displayed on top of the login webpage.

Reply:
Lets we keep this like this instead of adding a long command